storage/myisam/mi_check.c

```diff
@@ -335,21 +332,19 @@ int chk_size(HA_CHECK *param, register MI_INFO *info)
     {
       error=1;
       mi_check_print_error(param,
-			   "Size of indexfile is: %-8s        Should be: %s",
-			   llstr(size,buff), llstr(skr,buff2));
+        "Size of indexfile is: %-8lld        Should be: %lld", size, skr);
     }
     else
       mi_check_print_warning(param,
-			     "Size of indexfile is: %-8s      Should be: %s",
-			     llstr(size,buff), llstr(skr,buff2));
+        "Size of indexfile is: %-8lld      Should be: %lld", size, skr);
   }
   if (!(param->testflag & T_VERY_SILENT) &&
       ! (info->s->options & HA_OPTION_COMPRESS_RECORD) &&
       ulonglong2double(info->state->key_file_length) >
       ulonglong2double(info->s->base.margin_key_file_length)*0.9)
-    mi_check_print_warning(param,"Keyfile is almost full, %10s of %10s used",
-			   llstr(info->state->key_file_length,buff),
-			   llstr(info->s->base.max_key_file_length-1,buff));
+    mi_check_print_warning(param,
+      "Keyfile is almost full, %10lld of %10lld used",
+      info->state->key_file_length, info->s->base.max_key_file_length-1);
```

Comment:
I'll stop commenting on the width in `%lld`. I think it make sense to specify the width in
* status output printfs, that print many rows and we want numbers to be aligned
* a warning/error that can be reported many times (e.g. some link is broken) and we want all numbers there to be aligned

if it's a warning/error that is reported only once per file "file length is wrong", something in the header is wrong, etc — it doesn't need align to numbers, and there's nothing to align them with